root/etc/nftables.d/10-custom-filter-chains.nft

```diff
@@ -37,3 +36,24 @@
 #     ct state new log prefix "Firewall4 accepted egress: "
 # }
 
+## Uncomment the chains below to stage nftrace instrumentation
+## at the beginning of  both base chains filtering packets first
+## Example enables tracing of DNS packets to limit output volume
+## Adjust to Your requirements and observe output using
+## root@OpenWRT:~# nft monitor trace
+
+#     chain raw_prerouting {
+#     type filter hook prerouting priority raw; policy accept;
```

Comment:
Should this be `priority raw - 1` so that it catches before the default NoTrack chains that fw4 adds? So the trace starts at the very beginning.

Reply:
It would be necessary if one had multiple raw tables. Now I prepend rule to a single raw table we got to re-use existing hook.
3 main base tables change hook default action and require x - 1 additional hook.